index.html

```diff
@@ -0,0 +1,184 @@
+<h2 class="works-card-heading-large">Tonic</h2>
+<div class="works-card-client">
+    <p class="paragraph">CANOPY</p>
+    <p class="fa-solid fa-circle works-card-client-counter"></p>
+    <p class="works-card-client-role paragraph">Back End Dev</p>
+    <p class="fa-solid fa-circle works-card-client-counter"></p>
+    <p class="works-card-client-year paragraph">2015</p>
+</div>
+<p class="paragraph works-card-paragraph">
+    A daily selection of privately personalized reads; no accounts or sign-ups required.
+</p>
+<ul class="works-card-tags">
+    <li class="works-card-tag">html</li>
+    <li class="works-card-tag">css</li>
+    <li class="works-card-tag">javascript</li>
+</ul>
+<div>
+<button>See Project</button>
+</div>
+</section>
+<section class="works-card">
+<img src="./assets/Snapshoot Portfolio (1).png" alt="Recent Work">
+<h2 class="works-card-heading-large">Multi-Post<br>Stories</h2>
+<div class="works-card-client">
+    <p class="paragraph">CANOPY</p>
+    <p class="fa-solid fa-circle works-card-client-counter"></p>
+    <p class="works-card-client-role paragraph">Back End Dev</p>
+    <p class="fa-solid fa-circle works-card-client-counter"></p>
+    <p class="works-card-client-year paragraph">2015</p>
+</div>
+<p class="paragraph works-card-paragraph">
+    A daily selection of privately personalized reads; no accounts or sign-ups required.
+</p>
+<ul class="works-card-tags">
+    <li class="works-card-tag">html</li>
+    <li class="works-card-tag">css</li>
+    <li class="works-card-tag">javascript</li>
+</ul>
+<div>
+<button>See Project</button>
+</div>
+</section>
+<section class="works-card">
+<img src="./assets/Snapshoot Portfolio (2).png" alt="Recent Work">
+<h2 class="works-card-heading-large">Tonic</h2>
+<div class="works-card-client">
+    <p class="paragraph">CANOPY</p>
+    <p class="fa-solid fa-circle works-card-client-counter"></p>
+    <p class="works-card-client-role paragraph">Back End Dev</p>
+    <p class="fa-solid fa-circle works-card-client-counter"></p>
+    <p class="works-card-client-year paragraph">2015</p>
+</div>
+<p class="paragraph works-card-paragraph">
+    A daily selection of privately personalized reads; no accounts or sign-ups required.
+</p>
+<ul class="works-card-tags">
+    <li class="works-card-tag">html</li>
+    <li class="works-card-tag">css</li>
+    <li class="works-card-tag">javascript</li>
+</ul>
+<div>
+<button>See Project</button>
+</div>
+</section>
+<section class="works-card">
+<img src="./assets/Snapshoot Portfolio (3).png" alt="Recent Work">
+<h2 class="works-card-heading-large">Multi-Post<br>Stories</h2>
+<div class="works-card-client">
+    <p class="paragraph">CANOPY</p>
+    <p class="fa-solid fa-circle works-card-client-counter"></p>
+    <p class="works-card-client-role paragraph">Back End Dev</p>
+    <p class="fa-solid fa-circle works-card-client-counter"></p>
+    <p class="works-card-client-year paragraph">2015</p>
```

Comment:
- Kindly it should be better to use `<div> or <span >`, or   `ul` tag to group them, and `li` per icon instead of  `<p>` tag. The `<p> `tag is only used for paragraphs. please fix that to make your code looks more professional :+1:

Reply:
I edited it as you mentioned thanks for letting me know.